docs/containers/debug-common.md

```diff
@@ -109,8 +138,22 @@ Example `launch.json` configuration for debugging a .NET Core application:
 | `skipFiles` | Optional. Automatically skip files covered by these glob patterns. |
 | `trace` | Optional. Enable diagnostic output. |
 
+### python object properties
+
+| Property | Description | Default |
+| --- | --- | --- |
+| `host` | The host for remote debugging. | |
+| `port` | The port for remote debugging. | `5678` |
+| `pathMappings` | Maps the project path between local machine and remote host.  | |
+| `projectType` | Type of Python app. | |
+| `justMyCode` | Debug only user-written code. | |
+| `django` | Django debugging. | `false` |
+| `jinja` | Jinja template debugging (e.g. Flask). | `false` |
```

Comment:
Do not use "e.g." or "i.e." in docs - can you rephrase?

Reply:
updated this with (such as Flask)